kbase.yml

```diff
@@ -11,7 +11,7 @@ service-language:
     python
 
 module-version:
-    0.0.5
+    0.0.6
```

Comment:
Why is this version number different from the rest of them? Did the other versions get ignored on an update, and now we're just perpetuating a historical mistake? Can we just set the versions all to 0.0.6 in the other files as wel?